<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowServer</class>
 <widget class="QMainWindow" name="MainWindowServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>Click start server to start...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>315</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Server Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_serverPort">
           <property name="text">
            <string>30000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>start server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_stop">
         <property name="text">
          <string>stop server</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_record">
       <property name="text">
        <string>record</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>image</zorder>
   <zorder>btn_start</zorder>
   <zorder>btn_stop</zorder>
   <zorder>btn_record</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>lineEdit_serverPort</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
